tests/rest-api/methods.yml

```diff
@@ -203,6 +203,227 @@
         }
     ]
     }
+- uri: /v2/apps?embed=apps.taskStats
+  method: GET
+  content: |
+    {
+    "apps": [
+        {
+            "args": null,
```

Comment:
That should only be the case when you are producing incorrect JSON. (Which can easily happen when you modify `methods.yml`; at least, it keeps happening to me.)

I could successfully run your tests with the following, minimalistic JSON response:

```json
{
    "apps": [
        {
            "taskStats": {
                "startedAfterLastScaling": {
                    "stats": {
                      "counts": {
                          "staged": 0,
                          "running": 1,
                          "healthy": 1,
                          "unhealthy": 0
                      },
                      "lifeTime": {
                          "averageSeconds": 17024.575,
                          "medianSeconds": 17024.575
                      }
                    }
                }
            }
        }
    ]
    }
```